contracts/protocol/libraries/logics/DepositManagementLogic.sol

```diff
@@ -325,14 +324,26 @@ library DepositManagementLogic {
 
         ) = getTotalCollateralAmount(_user);
 
+        if (totalUsedCollateralInBaseCcy == 0) {
+            return (0, 0, 0);
+        }
+
         (uint256 collateralAmount, , ) = getCollateralAmount(_user, _liquidationCcy);
 
-        if (collateralAmount == 0) revert CollateralIsZero({ccy: _liquidationCcy});
+        if (collateralAmount == 0) {
+            return (0, 0, 0);
+        }
+
+        uint256 liquidationAmountInBaseCcy = 0;
+        uint256 coveredRatio = (totalCollateralInBaseCcy * Constants.PCT_DIGIT).div(
+            totalUsedCollateralInBaseCcy
+        );
 
-        uint256 liquidationAmountInBaseCcy = totalCollateralInBaseCcy * Constants.PCT_DIGIT >=
-            totalUsedCollateralInBaseCcy * Storage.slot().liquidationThresholdRate
-            ? 0
-            : totalUsedCollateralInBaseCcy.div(2);
+        if (coveredRatio < Storage.slot().fullLiquidationThresholdRate) {
```

Comment:
Should it not be > instead of <?

Reply:
`<` is correct because those threshold rates are like:
- fullLiquidationThresholdRate: 11765
- liquidationThresholdRate: 12500